tests/utils/test_env_checker.py

```diff
@@ -12,6 +12,9 @@ class ActionDictTestEnv(gym.Env):
     action_space = Dict({"position": Discrete(1), "velocity": Discrete(1)})
     observation_space = Box(low=-1.0, high=2.0, shape=(3,), dtype=np.float32)
 
+    def __init__(self, render_mode=None):
```

Comment:
Type hint

Reply:
(I know this might sound a bit nitpicky, especially since this is an internal piece of testing code, but it's helpful when you go back to this code after a 6 months long break and try to figure out what this was supposed to be)